src/main/webapp/style.css

```diff
@@ -116,3 +116,17 @@ h5 {
   line-height: .1;
   font-weight: bold;
 }
+*{
+  box-sizing: border-box;
+}
+.column {
+  float: left;
+  width: 33.33%;
+  padding: 5px;
+}
+/* Clearfix (clear floats) */
```

Comment:
What does this mean?

Reply:
I used a clearfix in order to line up the pictures side by side in the while inside of the card.